source/features/same-page-definition-jump.tsx

```diff
@@ -0,0 +1,27 @@
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import GitHubURL from '../github-helpers/github-url';
+
+function followLocalLink(event: delegate.Event<MouseEvent, HTMLAnchorElement>) {
```

Comment:
This causes other PR's test to fail 😅

![image](https://user-images.githubusercontent.com/44045911/91042174-ee187900-e643-11ea-85c3-8008b4b43c50.png)

Reply:
Wow I am shocked I forgot that.

Reply:
> Wow I am shocked I forgot that.

I am shocked this passed the test

Reply:
Master was not merged into this pr. I made PR enabling more lint rules.